app/models/user.rb

```diff
@@ -774,6 +774,12 @@ def comeback!
     save!(validate: false)
   end
 
+  def become_watcher(watchable)
+    return nil if watchable.watched_by?(self)
+
+    Watch.create!(user: self, watchable: watchable)
```

Comment:
```suggestion
    watches.create!(watchable:)
```

とかでいけるかもです。またもう少し工夫したらそもそも見ただけで何やってるか簡単にわかるようになって独自メソッドを追加しないで行けるようになるかも？ともおもいました〜。